### PR TITLE
Artifact Deployment through the catalog

### PR DESCRIPTION
**Pull request by baptistepa**
Added  a page that describes how you can deploy connectors trough the catalog.

### CODE REVIEW

File: user-guide/Cloud_Platform/Catalog/Deploying_A_DataMiner_Connector_to_your_system.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+uid: Deploying_A_DataMiner_Connector_to_your_system
+---
+
+# Deploying a DataMiner connector to your Cloud Connected system
+
+## Requirements to be able to use this feature
+
+First of all make sure that the DataMiner System you want to deploy to is connected to the cloud. You can find documentation about getting your system connected to the cloud [here](xref:Connecting_your_DataMiner_System_to_the_cloud).
+
+Be sure to check if you organization has been verified already. This is necessary so we can check for which connectors your organization has acquired licenses. If your organization is not verified you will not be able to deploy any connector to your system. Take a look at the [guide](xref:CloudConnectionVerification) for more info on how you can get your organization verified.
```
Comment (Nafalgar) [inline user-guide/Cloud_Platform/Catalog/Deploying_A_DataMiner_Connector_to_your_system.md:11]:
```suggestion
Be sure to check if your organization has been verified already. This is necessary so we can check for which connectors your organization has acquired licenses. If your organization is not verified you will not be able to deploy any connector to your system. Take a look at the [guide](xref:CloudConnectionVerification) for more info on how you can get your organization verified.
```

File: user-guide/Cloud_Platform/Catalog/Deploying_A_DataMiner_Connector_to_your_system.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+The last thing you need to do is link your DCP account to a user on the DataMiner. To do this you can follow [this](xref:Linking_your_DataMiner_and_DCP_account) guide.
```
Comment (Nafalgar) [inline user-guide/Cloud_Platform/Catalog/Deploying_A_DataMiner_Connector_to_your_system.md:13]: Rephrase "on the DataMiner"

File: user-guide/Cloud_Platform/Catalog/Deploying_A_DataMiner_Connector_to_your_system.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## How to deploy to your system
+
+To deploy a connector you must first find it in the Catalog. Once you have located your connecter you will be able to see a button that says "deploy".
```
Comment (Nafalgar) [inline user-guide/Cloud_Platform/Catalog/Deploying_A_DataMiner_Connector_to_your_system.md:17]:
```suggestion
To deploy a connector you must first find it in the Catalog. Once you have located your connector you will be able to see a button that says "deploy".
```

File: user-guide/Cloud_Platform/Catalog/Deploying_A_DataMiner_Connector_to_your_system.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+> [!NOTE]
+> You will only see the deploy button if your organization has a license for that specific connector. If your organization does not have a license the button will be greyed out and have the text "No License" in it. In that case we suggest you contact licensing@skyline.be.
+
+After clicking the deploy button you will be presented with a list of Cloud Connected systems for your organization that you can deploy the connector to. Select your target system(s) and confirm the deploy action. This will push the connector to your DataMiner System.
```
Comment (Sven-at-Skyline) [inline user-guide/Cloud_Platform/Catalog/Deploying_A_DataMiner_Connector_to_your_system.md:22]: Could also mention that pushing a connector will never automatically change the production version on the system

**Review by Nafalgar — APPROVED** [review]

**Comment by MariekeGO** [comment]
@baptistepa Could you check if everything is still OK for you after the review?

**Comment by baptistepa** [comment]
Looks good to me. Thanks for the review @MariekeGO.

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**